ipn/ipnserver/server.go

```diff
@@ -681,49 +680,22 @@ func tryWindowsAppDataMigration(logf logger.Logf, path string) string {
 
 // StateStore returns a StateStore from path.
 //
-// The path should be an absolute path to a file.
-//
-// Special cases:
-//
-//   * empty string means to use an in-memory store
-//   * if the string begins with "mem:", the suffix
-//     is ignored and an in-memory store is used.
-//   * if the string begins with "kube:", the suffix
-//     is a Kubernetes secret name
-//   * if the string begins with "arn:", the value is
-//     an AWS ARN for an SSM.
+// The path should correspond to a registered store in
+// ipn.NewStore.
```

Comment:
Restore some of the deleted docs so people have some examples of how to use this. You can refer to something else for more details.

Reply:
I removed this entire function to remove the confusion.